### ISSUE TITLE
Error on login integration but credential are correct

### ISSUE THREAD

**Issue by Scialla** [albertogeniola/meross-homeassistant#351]
**Describe the bug**
After upgrade from 1.21 (working) to latest 20200412.2, after restart HA, i have error because was unable to load integration

Removed existing integration, removed component from HACS, restarted HA, reinstalled component, restarted HA, is impossibile to find integration

Tried to clear cache, tried using many other browser, but nothing change, integration don't appear in search

Edited manifest,json file and added `"version": "20200412.2"`

Tried again after cache cleared and Integration appear, tried to configure it but, in config, i **_dont have_**:

**- HTTP Api Endpoint**
**- Skip MQTT certificate validation**

Tried only with correct mail and password (with i can log in meross app) and returned again prompt for login with error (!) whithout word

I can't login.....

**Your environment**
HomeAssistant version: 2022.5.3

Hardware environment: Rpi 3b

This is the error log:

```
Failed request to API. Response was: {'apiStatus': 0, 'sysStatus': 0, 'info': '', 'timeStamp': 1652098784, 'data': {'userid': '107633', 'email': 'xxxxx@me.com', 'key': '7920443c7755874ef44a0479984235f4', 'token': 'f548634eb4b6d9067afd1912c8ea0e7d6b552e20d6afeb60530be2e1090d50e0'}}
Failed request to API. Response was: {'apiStatus': 0, 'sysStatus': 0, 'info': '', 'timeStamp': 1652098809, 'data': {'userid': '107633', 'email': 'xxxxx@me.com', 'key': '7920443c7755874ef44a0479984235f4', 'token': 'f548634eb4b6d9067afd1912c8ea0e7d516a08ca0ce9abe096f79534ad75c6ad'}}
Failed request to API. Response was: {'apiStatus': 0, 'sysStatus': 0, 'info': '', 'timeStamp': 1652099028, 'data': {'userid': '107633', 'email': 'xxxxx@me.com', 'key': '7920443c7755874ef44a0479984235f4', 'token': 'f548634eb4b6d9067afd1912c8ea0e7ddaad6645700ef5972c6ea017c6b44fa5'}}
Failed request to API. Response was: {'apiStatus': 1002, 'sysStatus': 0, 'info': 'No such user', 'timeStamp': 1652099040, 'data': {}}
Failed request to API. Response was: {'apiStatus': 0, 'sysStatus': 0, 'info': '', 'timeStamp': 1652099689, 'data': {'userid': '107633', 'email': 'scialla@me.com', 'key': '7920443c7755874ef44a0479984235f4', 'token': 'f548634eb4b6d9067afd1912c8ea0e7d9ccb35e7e72107e2b38d5de8d8304cc3'}}
```
and
```
Error occurred when logging in.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/meross_iot/api.py", line 92, in _login
    response_data = self._authenticated_post(_LOGIN_URL, params_data=data)
  File "/usr/local/lib/python3.9/site-packages/meross_iot/api.py", line 75, in _authenticated_post
    raise AuthenticatedPostException("Failed request to API. Response was: %s" % str(jsondata))
meross_iot.api.AuthenticatedPostException: Failed request to API. Response was: {'apiStatus': 0, 'sysStatus': 0, 'info': '', 'timeStamp': 1652098727, 'data': {'userid': '107633', 'email': 'xxxxx@me.com', 'key': '7920443c7755874ef44a0479984235f4', 'token': 'f548634eb4b6d9067afd1912c8ea0e7dae1c7be728f550d5647321cbf21a9554'}}
```

What can i do?

**Comment by Scialla** [comment]
![image](https://user-images.githubusercontent.com/49528981/167416322-dfdc658a-790a-4655-9b04-b9c5578d8969.png)

And after tried login

![image](https://user-images.githubusercontent.com/49528981/167416455-d87e96ef-53f4-493a-a670-6f826e794f4c.png)

**Comment by Scialla** [comment]
Roll back to master don't change anything..

**Comment by albertogeniola** [comment]
Glad you solved it.

**Comment by Scialla** [comment]
Solved with roll back to 1.2.1, is normal?

**Comment by albertogeniola** [comment]
Hi,
Actually v1.2.1 is aligned to master, so as soon as you solved with 1.2.1, then it means it works with master.

You test it again, though.
